src/components/styles/StyledLink.tsx

```diff
@@ -12,6 +12,7 @@ const linkStyles = css`
   padding: 0;
   cursor: pointer;
   color: ${blue.base};
+  width: fit-content;
```

Comment:
What does this for?

Reply:
Would it make sense to apply this just to the link element we are targeting instead of making it universal to all links?